dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -159,6 +199,15 @@ std::pair<UInt64, UInt64> MinMaxIndex::getUInt64MinMax(size_t pack_index)
 RSResult MinMaxIndex::checkNullableEqual(size_t pack_index, const Field & value, const DataTypePtr & type)
 {
     const auto & column_nullable = static_cast<const ColumnNullable &>(*minmaxes);
+    const auto & null_map = column_nullable.getNullMapColumn();
+    bool min_is_null = null_map.getElement(pack_index * 2);
+
+    // if min value is null, then it must be the old version of minmax index.
```

Comment:
May be you could describe it as "prior to v6.x, the min max index may ...." instead of using "old", because "old" and "new" is not a good description after this PR merged into master. Consider the case, when you look at the code, you see some code describe "old" and "new" -- you don't even know how old the old is.. and this piece of code may exist even years ago.